src/credentials/DeviceAttestationVerifier.h

```diff
@@ -60,7 +60,9 @@ enum class AttestationVerificationResult : uint16_t
 
     kAttestationSignatureInvalid = 800,
 
-    kNoMemory = 900,
+    kAttestationElementsMalformed = 900,
+
+    kNoMemory = 1000,
```

Comment:
All error codes in range 500..1000 relate to internal validation of data. Suggest just putting all of them in the 500 range

Reply:
will update